### ISSUE TITLE
--unsuppress should be the default

### ISSUE THREAD

**Issue by joehillen** [aurapm/aura#339]
As a user, I had been confused when installing source packages why `aura` seemed to hang for a long time for seemingly no reason.

Turns out `aura` was just running `makepkg` but suppressing its output. This is a bad user experience when builds or downloads take a long time. `yaourt` doesn't hide this output, why should `aura`?

I propose the flag `--suppress` be added and `--unsuppress` be made the default behavior. The flag should stay in order to remain compatible with existing wrappers.

**Comment by fosskers** [comment]
`--unsuppress` is the default for exactly the reason you stated: yaourt doesn't suppress anything. I personally found yaourt overly verbose, and thus most messages are suppressed by design.

**Comment by joehillen** [comment]
The issue I had is that I have no idea what is going on during makepkg. Before I understood the install process, I would sit and scratch my head and assume the install was stuck for some reason or another and interrupt it. I did not realize that some package build from source and that can take a very long time. 

There is obviously value in having a `--suppress` flag for people like you, but as a new user, this behavior was frustrating because I did not know what was happening.

I think the interface should error on the side of verbose over frustrating.
